dynamo/vectorfield/vector_calculus.py

```diff
@@ -1109,53 +925,34 @@ def curl(adata, basis="umap", vector_field_class=None, method="analytical", **kw
 
 
 def divergence(
-    adata,
-    cell_idx=None,
-    sampling=None,
-    sample_ncells=1000,
-    basis="pca",
+    adata: AnnData,
+    cell_idx: bool = None,
```

Comment:
The description of this function misses a few Args, and the type of cell_idx/sampling is not bool. The same problem can be found on line 330. I recommend using` Optional[Literal[...]]` for parameters like sampling.